.grit/patterns/js/object_shorthand.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Object shorthand
+tags: [good-practice]
+---
+This matches the [eslint rule](https://eslint.org/docs/latest/rules/object-shorthand).
```

Comment:
It's okay to include a reference to the eslint rule, but you shouldn't need to click it to know what a rule actually does.

It's usually better to copy over the description.